apps/studio/components/interfaces/QueryPerformance/QueryPerformance.constants.ts

```diff
@@ -62,4 +62,8 @@ export const QUERY_PERFORMANCE_ROLE_DESCRIPTION = [
     description:
       'An internal role Supabase uses for administrative tasks, such as running upgrades and automations.',
   },
+  {
+    name: 'pgbouncer',
+    description: 'PgBouncer is a lightweight connection pooler for PostgreSQL. Paid plans only.',
```

Comment:
```suggestion
    description: 'PgBouncer is a lightweight connection pooler for PostgreSQL. Available on paid plans only.',
```

Maybe, just a little nicer?

Reply:
Could throw in dedicated
PgBouncer is a lightweight dedicated connection pooler for PostgreSQL.

No hard feelings whether to include the paid plan info at all here

Reply:
@kevcodez Totally mention paid plan, I was just wondering if to make it less blunt. Up to you , will approve anyway.

Reply:
Sorry, I went a bit "schnauze" there @ChrisChinchilla thanks for unbluntifying it! 🙇